doc/content/examples/DefiniteMaxwell.md

```diff
@@ -0,0 +1,53 @@
+# Definite Maxwell Problem
+
+## Summary
+
+Solves a 3D electromagnetic diffusion problem for
+  the electric field on a cube missing an octant, discretised using $H(\mathrm{curl})$ conforming Nédélec elements. This example
+  is based on [MFEM Example 3](https://mfem.org/examples/).
+
+
+## Description
+
+This problem solves a definite Maxwell equation with strong form:
+
+\begin{equation}
+\begin{split}
+\vec\nabla \times \vec\nabla \times \vec E + \vec E = \vec f \,\,\,&\mathrm{on}\,\, \Omega \\
+\vec E \times \hat n= \vec g \,\,\, &\mathrm{on}\,\, \partial\Omega
+\end{split}
+\end{equation}
+
+where
+
+\begin{equation}
+\vec f = \begin{pmatrix}
+    (1+\pi^2)\sin(\pi y)\\
+    (1+\pi^2)\sin(\pi z)\\
+    (1+\pi^2)\sin(\pi x)
+\end{pmatrix},\,\,\,
+\vec g = \begin{pmatrix}
+    \sin(\pi y)\\
+    \sin(\pi z)\\
+    \sin(\pi x)
+\end{pmatrix}
+\end{equation}
+
+In this example, we solve this using the weak form
+
+!equation
+-(\vec\nabla \times \vec E, \vec\nabla \times \vec v)_\Omega + (\vec E, \vec v)_\Omega
```

Comment:
Is there meant to be a negative sign here? [This random pdf](https://www.math.uci.edu/~chenlong/226/Maxwellweakform.pdf) I found doesn't have one (eq.1 vs eq.10)

Reply:
Nope - good spot - removed